ferret/modeling/text_helpers.py

```diff
@@ -124,7 +126,9 @@ def _forward(
         batches = np.array_split(text, n_batches)
 
         outputs = list()
+        max_len = 0
```

Comment:
I don't think we need this anymore, correct?

Reply:
Correct, thanks.